model/src/main/resources/reference.yml

```diff
@@ -152,6 +152,7 @@ ort:
       VulnerableCode:
         options:
           serverUrl: 'http://localhost:8000'
+          readTimeout: 40
```

Comment:
Maybe `readTimeoutSeconds`?

Reply:
I was considering `readTimeout*In*Seconds`, but then went for the shorter name as other timeouts that are configurable (FossID, ScanCode) also do not mention the unit.